src/GitLink.Tests/Providers/CustomRawUrlProviderFacts.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace GitLink.Tests.Providers
+{
+    using GitLink.Providers;
+    using NUnit.Framework;
+
+    public class CustomRawUrlProviderFacts
+    {
+        [TestFixture]
+        public class TheInitialization
+        {
+            [TestCase]
```

Comment:
It seemed like tests for other providers used explicit test case names for the positive and negative results so I kept that pattern. Let me know if you still like it changed.